certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/ag7804/CertificateModelFactoryAG7804.java

```diff
@@ -0,0 +1,189 @@
+        <li><b>Plan för återgång i arbete.</b> Sedan 2018 har arbetsgivare en skyldighet enligt 30 kap. 6 § socialförsäkringsbalken att ta fram en plan för återgång i arbete när en arbetstagare förväntas vara sjuk under en längre tid. Planen ska kontinuerligt följas upp och revideras vid behov. Planen är främst ett stöd för rehabiliteringsarbetet på arbetsplatsen och ska underlätta en tidig återgång i arbete. Läkarintyget ger grundläggande medicinsk information om patientens möjligheter att återgå i arbete och är en förutsättning för att arbetsgivaren tillsammans med sin arbetstagare ska kunna upprätta en plan för återgång i arbete.</li>
+        <li><b>Sjukpenningtillägg.</b> Att ge arbetstagaren rätt till kompletterande sjukpenningtillägg. De flesta kollektivavtal på svensk arbetsmarknad ger arbetstagare rätt till sjukpenningtillägg vid sjukdom.</li>
+        <li><b>Läkarintyg från och med dag 1 i sjuklöneperioden</b> Läkarintyget kan användas från och med dag 1 i sjuklöneperioden, ett så kallat förstadagsintyg. Arbetsgivaren får om det finns särskilda skäl för det begära att arbetstagaren genom intyg av läkare styrker nedsättning av arbetsförmågan tidigare än dag 8 i sjuklöneperioden.</li>
+      </ul>
+      <p><b>Innehåll i läkarintyget</b></p>
+      <p>Läkarintyg om arbetsförmåga - arbetsgivaren skapas från Försäkringskassans läkarintyg för sjukpenning (FK 7804) och ska innehålla motsvarande information.  Patienten kan dock välja om hen vill delge arbetsgivaren information om vilken eller vilka diagnoser som föreligger.  Övrig information i intyget är obligatorisk.</p>
+      <p><b>När ska Läkarintyg om arbetsförmåga – arbetsgivaren användas?</b></p>
+      <ul>
+        <li>När patienten har en anställning och behöver ett läkarintyg i förhållande till sin arbetsgivare.</li>
+        <li>När patienten varit frånvarande från arbetet i fler än 14 dagar.</li>
+        <li>När intygsutfärdande läkaren bedömer att patienten behöver vara sjukfrånvarande längre tid än 14 dagar och att sjukfrånvaron därmed kommer att överstiga sjuklöneperioden. Läkarintyg om arbetsförmåga – arbetsgivaren kan utfärdas för att undvika ett återbesök efter dag 14 och fyller då funktionen som intyg till arbetsgivare både under sjuklöneperioden och efter sjuklöneperioden.</li>
+      </ul>
+      """;
+
+  private static final String PREAMBLE_TEXT = """
+      Det här är ditt intyg. Intyget innehåller all information som vården fyllt i. Du kan inte ändra något i ditt intyg. Har du frågor kontaktar du den som skrivit ditt intyg. Det här intyget behöver du skriva ut och skicka själv.
+      """;
+
+  public static final CertificateModelId AG7804_V2_0 = CertificateModelId.builder()
+      .type(new CertificateType(AG_7804))
+      .version(new CertificateVersion(VERSION))
+      .build();
+
+  @Override
+  public CertificateModel create() {
```

Comment:
Even if we can't send the certificate we still need recipient for the pdf. Since you don't include send as an action it won't activate the send certificate resource link even if there is a recipient present.